volume/mounts/windows_parser.go

```diff
@@ -247,7 +248,11 @@ func (p *windowsParser) validateMountConfigReg(mnt *mount.Mount, additionalValid
 			return &errMountConfig{mnt, err}
 		}
 		if !exists {
-			return &errMountConfig{mnt, errBindSourceDoesNotExist(mnt.Source)}
+			if err := system.MkdirAllWithACL(mnt.Source, 0, system.SddlAdministratorsLocalSystem); err != nil {
```

Comment:
I think we're potentially missing a condition in the Windows implementation. When using the shorthand syntax (`-v <hostpath>:<containerpath>`), then the source-location should be created, but when using the long-form (`--mount type=bind,src=<foo>,dest=<foo>`) then the default is to not create the host-path.

I see the Linux code has a check for this, but I _guess_ the Windows code didn't have this, because it would never create the host-path; https://github.com/moby/moby/blob/7ced56b35627c2cc1e43a97d3afe858f6cfaa64f/volume/mounts/linux_parser.go#L83-L91